README.md

```diff
@@ -70,43 +82,26 @@ bash cluster-startup.sh NVLINK
 
 ## Running the Queries
 
-To run a query, starting from the repository root, go to the `queries` directory:
+To run a query, starting from the repository root, go to the query specific subdirectory. For example, to run q07:
 
 ```bash
-cd tpcx_bb/queries/
+cd tpcx_bb/queries/q07/
 ```
 
-Choose a query to run, and `cd` to that directory. We'll pick query 07.
+The queries assume that they can attach to a running Dask cluster. Cluster address and other benchmark configuration lives in a yaml file.
 
 ```bash
-cd q07
-```
-
-Activate the conda environment you've created.
-
-The queries assume that they can attach to a running Dask cluster. Command line arguments are used to determine the cluster and dataset on which to run the queries. The following is an example of running query 07.
-
-```bash
-python tpcx_bb_query_07.py --data_dir=$DATA_DIR --cluster_host=$SCHEDULER_IP --output_dir=$OUTPUT_DIR
+conda activate rapids-tpcx-bb
+python tpcx_bb_query_07.py --config_file=../../benchmark_runner/benchmark_config.yaml
 ```
 
-- `data_dir` points to the location of the data
-- `cluster_host` corresponds to the schedyler address of the running Dask cluster
-    - In this case, this query would attempt to connect to a cluster running at `$SCHEDULER_IP`, which we configured during cluster startup
-- `output_dir` points to where the queries should write output files. If not specified, queries will write output into their respective directories
-
-
 ## Performance Tracking
 
-This repository contains support for performance-tracking automation using Google Sheets. If you install `gspread` and `oauth2client` in the provided environment spec, passing `--sheet` and `--tab` arguments during query execution will log query runtime in a Google Sheet. You must provide a path to a Google Sheets API credentials file as an environment variable called `GOOGLE_SHEETS_CREDENTIALS_PATH` on the node running the client. If you do not have these libraries installed, you will see `Please install gspread and oauth2client to use Google Sheets automation` after running a query. It's fine to ignore this output.
-
-Expanding on the example above, you could do the following:
-
-```bash
-python tpcx_bb_query_07.py --data_dir=$DATA_DIR --cluster_host=$SCHEDULER_IP --output_dir=$OUTPUT_DIR --sheet=TPCx-BB --tab="SF1000 Benchmarking Matrix"
+This repository contains support for performance-tracking automation using Google Sheets. To enable logging query runtimes:
 ```
-
-
+conda install -c conda-forge gspread oauth2client
```

Comment:
This is probably fine, but the recommended installation process is with pip from the documentation of these libraries.

Reply:
moved into the conda env file